docs/api/datamodules.rst

```diff
@@ -131,6 +131,11 @@ SSL4EO
 .. autoclass:: SSL4EOLDataModule
 .. autoclass:: SSL4EOS12DataModule
 
+SSL4EOLBenchmarkDataModule
```

Comment:
Should be human-readable name, not class name. Like "SSL4EO-L Benchmark". Same in `datasets.rst` and in `non_geo_datasets.csv`